roles/splunk_cluster_master/tasks/main.yml

```diff
@@ -35,7 +35,7 @@
   when: splunk.smartstore
 
 - name: Set the current node as a Splunk indexer cluster master
-  command: "{{ splunk.exec }} edit cluster-config -mode master -replication_factor {{ splunk.idxc.replication_factor }} -search_factor {{ splunk.idxc.search_factor }} -secret '{{ splunk.idxc.secret }}' -cluster_label '{{ splunk.idxc.label }}' -auth 'admin:{{ splunk.password }}'"
```

Comment:
In fact, could we not define idxc_replication_factor and idxc_search_factor up in this playbook since they are just redefined values of splunk.idx.{{variable}}.

Reply:
When num_indexer_hosts < 3, they will be set to 1.